cheetah/utils.py

```diff
@@ -109,7 +108,7 @@ def ocelot2cheetah(element, warnings=True):
         return acc.VerticalCorrector(element.l, element.angle, name=element.id)
     elif isinstance(element, oc.Cavity):
         return acc.Cavity(element.l, name=element.id)
-    elif isinstance(element, oc.Monitor) and "BSC" in element.id:
+    elif isinstance(element, oc.Monitor) and "BSC" in element.id or "SCR" in element.id:
```

Comment:
A little thing here @FelixTheilen ... can you check precedence of `and` and `or`. I worry than as it is now, if `"SCR`" is in the name, it will completely ignore whether it is an instance of `oc.Monitor`.

Reply:
It works in the desired way. `Drifts` containing `"SCR"` are not converted to `Screens`. However I added parentheses, as they will make the code more readable.